frontend_new/src/themes/light.tsx

```diff
@@ -3,13 +3,19 @@ import { createMuiTheme } from "@material-ui/core/styles";
 export const light = createMuiTheme({
   palette: {
     primary: {
-      main: "#f0f0f0",
+      // main: "#f0f0f0",
+      main: "#122542",
+      light: "#F3F3F1",
     },
     secondary: {
-      main: "#212121",
+      //main: "#212121",
+      main: "#F19828",
+      
     },
+    
+    
     background: {
-      default: "#dfdfdf",
+      default: "#f7f7f5",
```

Comment:
**[Desired,Maintainability]** We should probably have a standalone file where we store all the global styling (main page elements). Having constants inline all over the code base is a maintainability nightmare.